alacritty/src/display/mod.rs

```diff
@@ -476,11 +477,13 @@ impl Display {
 
         #[allow(clippy::single_match)]
         #[cfg(not(windows))]
-        match config.window.startup_mode {
-            #[cfg(target_os = "macos")]
-            StartupMode::SimpleFullscreen => window.set_simple_fullscreen(true),
-            StartupMode::Maximized if !is_wayland => window.set_maximized(true),
-            _ => (),
+        if !_tabbed {
```

Comment:
Can I ask why this variable is prefixed with an underscore and used? Seems like a small oversight.